spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/route/Route.java

```diff
@@ -130,6 +133,10 @@ public String toString() {
 		return sb.toString();
 	}
 
+	public TimeoutConfig getTimeout() {
```

Comment:
Things have been updated recently. Let's move this to use the new metadata map with specific keys rather than updating the

Reply:
reworked using metadata